OpenTrons/opentrons-api
# fix(protocol-designer): remove fixed trash option from dropdowns

closes [RAUT-908](https://opentrons.atlassian.net/browse/RAUT-908)

# Overview

In PD schemaV8, we use trash bin entities on OT-2 rather than implictly loaded fixed trash single-well labwares. Here, I update the v8 migration to remove loadLabware command for fixedTrash so that it is not present as an option alongside Trash Bin in dropdowns for dispense, droptip, etc.

# Test Plan

- import an old version of PD protocol ([like this v3 protocol](https://github.com/Opentrons/opentrons/files/13720048/doItAllV3.json))
- in protocol design, select a step that accepts trash locations (dispense, blowout, droptip...)
- observe that 'Fixed Trash' is no longer present. Only 'Trash Bin' should be present (refers to OT-2 slot 12 trash addressable area) 

<img width="502" alt="Screen Shot 2023-12-19 at 3 27 55 PM" src="https://github.com/Opentrons/opentrons/assets/47604184/ff7d0dbc-26ea-4d41-8cbf-4f9fbd8b9180">

# Changelog

- filter loadLabware command specific to fixedTrash in v8 migration file

[RAUT-908]: https://opentrons.atlassian.net/browse/RAUT-908?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
